docs/debugging.rst

```diff
@@ -0,0 +1,97 @@
+.. _debugging:
+
+Piko/RT Debugging
+=================
+
+
+Using tmux
+----------
+
+`tmux <https://github.com/tmux/tmux>`_ is a "terminal multiplexer", it allow you
+accessed and controllerd multiple terminals inside a signle terminal like this:
+
+.. image:: https://gifyu.com/images/asciicast.json.gif
+
+How to use tmux, please reference these pages:
+
+* `tmux wiki <https://github.com/tmux/tmux/wiki>`_
+* `A tmux Crash Course <https://robots.thoughtbot.com/a-tmux-crash-course>`_
+* `A Quick and Easy Guide to tmux <http://www.hamvocke.com/blog/a-quick-and-easy-guide-to-tmux/>`_
+
+Using OpenOCD
+-------------
+
+XXX
+
+Change CFLAGS_OPT to build with debugging information
+-----------------------------------------------------
+
+You can edit ``mk/flags.mk`` to change the ``CFLAGS_OPT`` to include debugging
+information to let you debug more easily, for example, change ``CFLAGS_OPT`` from::
+
+   CFLAGS_OPT = -Os # -flto
+
+to this::
+
+   CFLAGS_OPT = -O0 -g3 -gdwarf-4
```

Comment:
We might introduce debug build, which generates debugging symbols.

Reply:
Drop this in docs.